packages/app/features/DashboardLayout/Navigator.tsx

```diff
@@ -40,10 +41,26 @@ export const DashboardNavigator: FC<Props> = ({
 		} as never,
 	};
 	const isExplorerActive = getIsExtensionActive?.(exploreItem as never);
+	const { isMobile } = runtime;
```

Comment:
no platform-conditional here, please revise this with me @dev-tronghieu

Reply:
as I use the same `DashboardNavigator` component for both mobile and extension and their UIs are not identical, I have to check for the platform to manage these differences